### PR TITLE
Update species compendium section

### LINKED ISSUE #70: Update species compendia section
https://github.com/AlexsLemonade/refinebio/issues/508#issuecomment-435879283

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
Closes #70. Updates our species compendium section to include current processing steps and an additional diagram.

### CODE REVIEW

File: docs/main_text.md [hunk 1]
```diff
@@ -362,20 +362,27 @@ Experiment metadata (e.g., experiment description and title) is delivered in the
 
 # Species compendia
 
-refine.bio is currently in beta.
-Once refine.bio reaches production, we will periodically release compendia comprised of all the samples from a species that we were able to process.
+We plan to periodically release compendia comprised of all the samples from a species that we were able to process.
```
Comment (cgreene) [inline docs/main_text.md]: We will ...

Comment (cgreene) [reply docs/main_text.md]: Or just... "We periodically..."

File: docs/main_text.md [hunk 2]
```diff
@@ -362,20 +362,27 @@ Experiment metadata (e.g., experiment description and title) is delivered in the
...
 We refer to these as **species compendia**.
-We'll process these compendia in a manner that is different from the options that are available via the web user interface.
-We describe our intended processing pipeline below.
-Instead of selecting only genes available in all samples, we take the union of all genes, filling in any missing values with `NA` (e.g., perform a full outer join as illustrated below).
+We process these compendia in a manner that is different from the options that are available via the web user interface.
```
Comment (cgreene) [inline docs/main_text.md]: We should say why here. Something like:

> These species compendia provide a snapshot of the most complete collection of gene expression data that refine.bio can produce for each supported organism.

File: docs/main_text.md [hunk 3]
```diff
@@ -362,20 +362,27 @@ Experiment metadata (e.g., experiment description and title) is delivered in the
...
 # Species compendia
 
-refine.bio is currently in beta.
-Once refine.bio reaches production, we will periodically release compendia comprised of all the samples from a species that we were able to process.
+We periodically release compendia comprised of all the samples from a species that we were able to process.
 We refer to these as **species compendia**.
-We'll process these compendia in a manner that is different from the options that are available via the web user interface.
-We describe our intended processing pipeline below.
-Instead of selecting only genes available in all samples, we take the union of all genes, filling in any missing values with `NA` (e.g., perform a full outer join as illustrated below).
+We process these compendia in a manner that is different from the options that are available via the web user interface.
```
Comment (cgreene) [inline docs/main_text.md]:
```suggestion
We process these compendia in a manner that is different from the options that are available via the web user interface.
These species compendia provide a snapshot of the most complete collection of gene expression that refine.bio can produce for each supported organism.
```

File: docs/main_text.md [hunk 4]
```diff
@@ -362,20 +362,27 @@ Experiment metadata (e.g., experiment description and title) is delivered in the
...
+
+Instead of selecting only genes available in all samples, we take the union of all genes, filling in any missing values with `NA`.
```
Comment (cgreene) [inline docs/main_text.md]:
```suggestion
The refine.bio web interface does an inner join when datasets are combined, so only genes present in all datasets are included in the final matrix.
For compendia, we take the union of all genes, filling in any missing values with `NA`. 
```

File: docs/main_text.md [hunk 5]
```diff
@@ -362,20 +362,27 @@ Experiment metadata (e.g., experiment description and title) is delivered in the
...
 
+We perform an outer join each time samples are combined in the process of building species compendia.
+
+![docs-species-compendia](https://user-images.githubusercontent.com/15315514/48498088-72995f00-e803-11e8-8832-3a9024748431.png)
 
-We drop any genes that have missing values in greater than 30% of samples.
-We impute the remaining missing values with KNN impute.
+Samples from each technology—microarray and RNA-seq—are combined separately.
+In RNA-seq samples, we filter out genes with low total counts and then `log2(x + 1)` the data.
+We join samples from both technologies, drop genes that have missing values in greater than 30% of samples, and drop samples that have missing values in greater than 50% of genes.
```
Comment (cgreene) [inline docs/main_text.md]: Just to check, you drop samples first, then genes?

Comment (cgreene) [inline docs/main_text.md]:
```suggestion
We join samples from both technologies.
We then drop genes that have missing values in greater than 30% of samples.
Finally, we drop samples that have missing values in greater than 50% of genes.
```

Comment (jaclyn-taroni) [reply docs/main_text.md]: No, we drop genes (https://github.com/AlexsLemonade/refinebio/blob/a27543a83ebe6946fe87ed082a64458ad7d57b12/workers/data_refinery_workers/processors/create_compendia.py#L137) and then samples (https://github.com/AlexsLemonade/refinebio/blob/a27543a83ebe6946fe87ed082a64458ad7d57b12/workers/data_refinery_workers/processors/create_compendia.py#L145). I'll include your suggested change below.

File: docs/main_text.md [hunk 6]
```diff
@@ -362,20 +362,31 @@ Experiment metadata (e.g., experiment description and title) is delivered in the
...
 We refer to these as **species compendia**.
-We'll process these compendia in a manner that is different from the options that are available via the web user interface.
-We describe our intended processing pipeline below.
-Instead of selecting only genes available in all samples, we take the union of all genes, filling in any missing values with `NA` (e.g., perform a full outer join as illustrated below).
+We process these compendia in a manner that is different from the options that are available via the web user interface.
+These species compendia provide a snapshot of the most complete collection of gene expression that refine.bio can produce for each supported organism.
+
+The refine.bio web interface does an inner join when datasets are combined, so only genes present in all datasets are included in the final matrix.
+For compendia, we take the union of all genes, filling in any missing values with `NA`. 
+This is a "full outer join" as illustrated below.
```
Comment (dvenprasad) [inline docs/main_text.md]: Would it make sense to add rationale for choosing outer join? (It's not obvious to me but could be to researchers)

Comment (jaclyn-taroni) [reply docs/main_text.md]: I believe it is likely to be obvious to folks that are most interested in this offering, but that's a hunch based on my experience. I'm happy to change it if that turns out not to be the case.

**Pull request merged**